storybook/src/pages/amsterdam/article/Article.docs.mdx

```diff
@@ -0,0 +1,66 @@
+{/* @license CC0-1.0 */}
+
+import { Meta } from "@storybook/blocks";
+import * as ArticleStories from "./Article.stories.tsx";
+
+<Meta of={ArticleStories} />
+
+# Article Page
+
+This page type is mainly for news or related kinds of articles.
+
+## Layout
+
+```tsx
+import { Breadcrumb } from "./Breadcrumb";
+<SkipLink href="#main">Direct naar inhoud</SkipLink>
+<Screen maxWidth="wide">
+  <Grid>
+    <Header />
+  </Grid>
+  <main id="main">
+    <Grid>
+      <Breadcrumb/>
+      <Heading>
+      <Paragraph size="small" />
+    </Grid>
+    <AspectRatio>
+      <Image />
+    </AspectRatio>
+    <Grid>
+      <Paragraph size="large" />
+      <Paragraph />
+      <Heading level={2} />
+      <Paragraph />
+      <OrderedList />
+      <Paragraph />
+      <Heading level={3} />
+      <Paragraph />
+      <Accordion />
+      <Paragraph />
+    </Grid>
+    <Spotlight>
+      <Grid>
+        <Newsletter />
+        <Share />
+      </Grid>
+    </Spotlight>
+    <Grid>
+      <RelatedArticles />
+    </Grid>
+  </main>
+  <Footer>
+    <Grid />
+  </Footer>
+</Screen>
+```
+
+1. The page for an article starts with the BreadCrumb.
+2. The title is displayed as a level 1 Heading.
+3. Below that, the publication date and any tags.
+4. The main image for the article takes the full width.
+5. The lead paragraph uses Text Level 4.
+6. Article Headings have level 2.
+7. Sections can be wrapped in an Accordion.
```

Comment:
This is a little confusing. Do you mean: 'You can use an Accordion'? I'm not sure we need to specify that.